src/main/java/org/deepsymmetry/beatlink/Util.java

```diff
@@ -132,6 +140,13 @@ public enum PacketType {
          */
         CDJ_STATUS(0x0a, "CDJ Status", VirtualCdj.UPDATE_PORT),
 
+        /**
+         * Metadata that includes track album art, possibly waveforms and more. We do not use this information at the moment
```

Comment:
I’d leave out the speculation about waveforms; there is no evidence they are present, and there is no theoretical reason they should be since they are not displayed in the rekordbox UI.